addons/csw/XEH_postInit.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+
+if (isServer) then {
+    [QGVAR(addObjectToServer), {
+        params["_object"];
+        _object setOwner 2;
```

Comment:
It's not immedially clear that `2` is the server's owner id. Could you change this to be a defined macro?

```sqf
#define SERVER_OWNER_ID 2

....
[QGVAR(addObjectToServer), {
     params ["_object"];
    _object setOwner SERVER_OWNER_ID;
    }] call CBA_fnc_addEventHandler;
```